app/models/project.rb

```diff
@@ -18,4 +18,12 @@ def to_param
   def volunteer_emails
     self.volunteered_users.collect { |u| u.email }
   end
+
+  def volunteered_users_count
+    volunteered_users.count
+  end
+
+  def serializable_hash(options={})
+    super(methods: [:to_param, :volunteered_users_count, :project_type_list, :skill_list])
```

Comment:
Should be safe to expose all fields at this point, but just a headsup that sensitive fields in the future would need to be added to a blocklist here.